gadopt/scalar_equation.py

```diff
@@ -32,24 +32,33 @@ def residual(self, test, trial, trial_lagged, fields, bcs):
         if 'advective_velocity_scaling' in fields:
             u = fields['advective_velocity_scaling'] * u
 
-        F = -q*div(phi*u)*self.dx
+        if 'su_nubar' in fields:
+            # SU(PG) ala Donea & Huerta 2003
+            log("Using SU advection")
```

Comment:
I guess I understand the value of logging this, particularly for debugging? But because it's the same message as the one in the energy solver, it's a little trickier to figure out which term it's coming from. But I guess we don't actually know that at this level...

At least, I think maybe we should `from .utility import log_level` at the top of this file, and then conditionally print it:
```suggestion
            if INFO >= log_level:
                log("Using SU advection")
```

Reply:
I've removed this as I don't think it is really needed.